README.md

```diff
@@ -88,6 +88,7 @@ auto storage = make_storage("db.sqlite",
                             make_table("user_types",
                                        make_column("id", &UserType::id, autoincrement(), primary_key()),
                                        make_column("name", &UserType::name, default_value("name_placeholder"))));
+storage.sync_schema();
```

Comment:
why? You make the first block more complicated. Info about `sync_schema` is noted later and it is ok

Reply:
When I first used this library，I followed the README from top to bottom. before I read the `Migrations functionality`, my code was like:
```cpp
struct User{
    int id;
    std::string firstName;
    std::string lastName;
    int birthDate;
    std::unique_ptr<std::string> imageUrl;
    int typeId;
};

struct UserType {
    int id;
    std::string name;
};

int main(){
  using namespace sqlite_orm;
  auto storage = make_storage("db.sqlite",
                              make_table("users",
                                         make_column("id", &User::id, autoincrement(), primary_key()),
                                         make_column("first_name", &User::firstName),
                                         make_column("last_name", &User::lastName),
                                         make_column("birth_date", &User::birthDate),
                                         make_column("image_url", &User::imageUrl),
                                         make_column("type_id", &User::typeId)),
                              make_table("user_types",
                                         make_column("id", &UserType::id, autoincrement(), primary_key()),
                                         make_column("name", &UserType::name, default_value("name_placeholder"))));
  
  User user{-1, "Jonh", "Doe", 664416000, std::make_unique<std::string>("url_to_heaven"), 3 };
      
  auto insertedId = storage.insert(user);
  cout << "insertedId = " << insertedId << endl;      //  insertedId = 8
  user.id = insertedId;
  
  return 0;
}
```
But it didn't work because of no `sync_schema()` and I get an exception which took me a lot of time to debug. Maybe other developers would make the same mistake if we don't add this line at the first code section.

Reply:
Sounds logic but it is not acceptable to everyone. Some people don't use `sync_schema` but just use already existing database with correct schema